.github/workflows/validate-documentation.yml

```
@@ -0,0 +1,221 @@
+          # Install lychee link checker
+          wget -qO- https://github.com/lycheeverse/lychee/releases/latest/download/lychee-x86_64-unknown-linux-gnu.tar.gz | tar -xzv
+          sudo mv lychee /usr/local/bin/
+          
+      - name: Check for broken internal links
+        run: |
+          echo "Checking for broken internal links..."
+          
+          # Check internal markdown links
+          find manuals -name "*.md" -type f | while read file; do
+            echo "Checking internal links in $file..."
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)